package.json

```diff
@@ -26,28 +25,21 @@
     "url": "https://github.com/astarte-platform/astarte-dashboard"
   },
   "scripts": {
-    "start": "webpack serve --hot --inline --config webpack.dev.js",
-    "prebuild": "rimraf dist",
-    "build": "webpack --config webpack.prod.js",
-    "deploy": "webpack --config webpack.prod.js",
+    "start": "PORT=8080 react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "deploy": "npm run build",
     "lint": "eslint --fix src --ext .js,.jsx,.ts,.tsx",
     "check-lint": "eslint src --ext .js,.jsx,.ts,.tsx",
     "format": "prettier --write src",
-    "check-format": "npx prettier --check src",
+    "check-format": "prettier --check src",
     "check-types": "tsc --noEmit",
     "check-types:watch": "npm run check-types -- --watch",
     "cypress:open": "cypress open",
-    "test": "cypress run"
+    "cypress": "cypress run"
```

Comment:
we have `cypress:open` already, this could be `cypress:test`

Reply:
One that I've seen is `npm run cy:run`, so we could go with `npm run cypress:run`. But it seems redundant.
Another approach is to have a simple main command, and some :modifiers if necessary
```sh
npm run test                # launch jest
npm run test:watch      # launch jest in watch mode
npm run cypress          # launch cypress
npm run cypress:open # launch cypress in interactive mode
```
What do you think?

Reply:
i think that `run test` is ambiguous as both jest ad cypress are testing suites

i'd use
```
npm run jest:test
npm run jest:watch
npm run cy:test
npm run cy:open
```

Reply:
I went with
```
npm run jest:test
npm run cypress:test
npm run cypress:open
```
so it's definitely clear what's happening